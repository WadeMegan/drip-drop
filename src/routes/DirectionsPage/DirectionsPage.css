.directions-section h2{
    padding-bottom: 15px;
  }

  .directionsSection{
    padding-top:6vh;
  }

.directionsSection h3{
    font-size:22px;
    line-height:26px;
}