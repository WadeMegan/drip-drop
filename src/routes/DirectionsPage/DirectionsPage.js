import React, { Component } from 'react'
import { Link } from 'react-router-dom'
import './DirectionsPage.css'
import PlantItem from '../../components/PlantItem/PlantItem'
import PlantListContext from '../../contexts/PlantListContext'
import PlantApiService from '../../services/plant-api-services'
import UserService from '../../services/user-service'
import Error from '../../components/Error/Error'

export default class DirectionsPage extends Component {    
    render() {

        return (
            <Error>
            <div className='switchPagesNav'>
                <div className='pageButton directionsButton current'>
                    <p className='buttonItem'>Directions</p>
                </div>
                <Link className='pageButton availableButton notCurrent' to='plants'>
                    <p className='buttonItem'>Available Plants</p>
                </Link>
                <Link className='pageButton yourPlantsButton notCurrent' to='your-plants'>
                    Your Plants
                </Link>
            </div>
            <section className='directionsSection'>
                <div className='banner'>
                    <h2>Directions</h2>
    
                </div>
                <div className='all-plants-container'>

                </div>
            </section>
            </Error>
        )
    }

}