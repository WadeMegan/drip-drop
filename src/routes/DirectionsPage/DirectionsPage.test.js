import React from 'react'
import ReactDOM from 'react-dom'
import DirectionsPage from './DirectionsPage'
import {BrowserRouter} from 'react-router-dom'
import renderer from 'react-test-renderer'

it('renders without crashing',()=>{
    const div = document.createElement('div')
    ReactDOM.render(<BrowserRouter><DirectionsPage/></BrowserRouter>,div)
    ReactDOM.unmountComponentAtNode(div)
})

it('should match with snapshot', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <DirectionsPage/>
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot()
})
