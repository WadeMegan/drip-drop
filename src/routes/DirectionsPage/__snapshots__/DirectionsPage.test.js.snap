// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match with snapshot 1`] = `
Array [
  <div
    className="switchPagesNav"
  >
    <div
      className="pageButton directionsButton current"
    >
      <p
        className="buttonItem"
      >
        Directions
      </p>
    </div>
    <a
      className="pageButton availableButton notCurrent"
      href="/plants"
      onClick={[Function]}
    >
      <p
        className="buttonItem"
      >
        Available Plants
      </p>
    </a>
    <a
      className="pageButton yourPlantsButton notCurrent"
      href="/your-plants"
      onClick={[Function]}
    >
      Your Plants
    </a>
  </div>,
  <section
    className="directionsSection"
  >
    <div
      className="banner"
    >
      <h2>
        Directions
      </h2>
    </div>
    <div
      className="directionsContainer"
    >
      <div
        className="directionsSection"
        id="directionsSection"
      >
        <h3>
          Drip Drop makes watering your plants easy... Here's how it works.
        </h3>
        <div
          className="directionsText"
        >
          <p>
            <span
              className="step"
            >
              Step 1
            </span>
             The "Your Plants" page contains the plants that Drip Drop will text you about. To select plants, head on over to the "Available Plants" page and add any that you would like to be reminded to water. When you add a plant, you will begin receiving reminders for it the next day.
          </p>
          <p>
            <span
              className="step"
            >
              Step 2
            </span>
             Each day, Drip Drop will check if you have any plants that need to be watered. If so, you'll get a reminder text. To stop getting notifications for a plant, simply remove it.
          </p>
          <p
            className="step"
          >
            And that's it!
          </p>
        </div>
      </div>
    </div>
  </section>,
]
`;
